app/controllers/hooks_controller.rb

```diff
@@ -8,7 +8,7 @@ def create
     when 'issues', 'issue_comment'
       SyncSubjectWorker.perform_async_if_configured(payload['issue'])
     when 'pull_request', 'pull_request_review', 'pull_request_review_comment'
-      SyncSubjectWorker.perform_async_if_configured(payload['pull_request'])
```

Comment:
I think it'd be good tosplit `'pull_request'` out from the review and review_comment section, so it still calls `SyncSubjectWorker` here

Reply:
will do now

Reply:
I've pushed this but: it'll still sync the comments for the PR as a whole. I could split it out to check for only reviews but I'm not sure there's that much additional work to call the comments on the PR too... Up to you if you want it further split. I'm easy either way.